src/libraries/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -212,6 +212,9 @@
   <data name="net_auth_eof" xml:space="preserve">
     <value>Authentication failed because the remote party has closed the transport stream.</value>
   </data>
+  <data name="net_auth_tls_alert" xml:space="preserve">
+    <value>Authentication failed because the remote party has sent TLS alert {0}.</value>
```

Comment:
I recommend putting single quotes around the {0}.

Thus the error message would then look something like this:

"Authentication failed because the remote party has sent TLS alert 'Protocol Version'."